lib/message.rb

```diff
@@ -0,0 +1,25 @@
+class Message
+  def self.welcome
+    "\n\n-----------------------------------------------------\nWelcome To Tic Tac Toe\n-----------------------------------------------------"
+  end
+
+  def self.instruction
+    "\n\nPlease pick a spot by pressing one number in the range of 1-9\n\n"
+  end
+
+  def self.mode_select
+    "\nPlease select a game mode:\n 1. Human vs Human\n Human vs Computer\n"
+  end
```

Comment:
While you know there is an upcoming feature to add a second game mode, (HvC) generally it's best to treat each feature completely independently.  

This is important for a couple of reasons - from a user viewpoint, think about how it might look if you were to release this initial implementation to the public and it had references to a game mode that you couldn't actual play yet. That would be rather frustrating. From a developer viewpoint, when code for different features gets added all in one PR, it can be much harder to track down any bugs that may occur.

Reply:
Likewise you have a couple of empty files in this PR - typically you don't want to over optimize your code too early. So if these files aren't yet being used - I would argue they shouldn't be included in this PR.

A good practice I like to follow, is to closely review the ticket criteria when putting up the PR, and seeing if the code you have written adheres to the ticket's requirements. At some point soon we will get into the Single Responsibility Principle, which can also kinda apply to tickets/PRs.